yarn.lock

```diff
@@ -4730,6 +4730,13 @@ detect-port-alt@1.1.5:
     address "^1.0.1"
     debug "^2.6.0"
 
+detect-port-alt@1.1.6:
```

Comment:
wat?

Reply:
It's used by react-dev-utils, what's the problem with it?